### PR TITLE
Privacy -> private typo

### PR DESCRIPTION
**Pull request by zackisaacs**
Developers are pointing out the typos! Good stuff. I also let everyone know they can submit PRs for this stuff themselves 😜

### CODE REVIEW

File: content/docs/webhooks.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ These are the fields that are returned with webhook events:
       </span>
     </td>
     <td>
-        For update events, keys can contain "title," "type," and "privacy." Empty for delete and create events.
+        For update events, keys can contain "title," "type," and "private," which is always "true" or "false." Empty for delete and create events.
```
Comment (MateoAO) [inline content/docs/webhooks.md]: https://www.merriam-webster.com/dictionary/typo

This feels redundant
"For update events [snip] Empty for delete and create events."

My Suggestion:
For update events, keys can contain "title," "type," and "private".

Comment (zackisaacs) [reply content/docs/webhooks.md]: Alright I guess it was a _mistake_ and not a _typo_ -- excuse me!

I agree about redundancy. What about the "true" or "false" part? I think I like specifying that it's a boolean in this way.

Comment (MateoAO) [reply content/docs/webhooks.md]: Isn't that specified elsewhere?

Comment (zackisaacs) [reply content/docs/webhooks.md]: No. I think it makes sense to go here because this is where we are defining the field.

**Pull request merged**